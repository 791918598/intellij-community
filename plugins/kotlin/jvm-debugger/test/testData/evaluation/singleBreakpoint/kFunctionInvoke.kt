fun main() {
    //Breakpoint!
    val x = 1
}

inline fun Int.extInlineGenericFun() = this

fun notUsedFun() {
    val newName = ::someFun
    val ret = newName()
}

fun someFun() = 1

// EXPRESSION: 42.extInlineGenericFun()
// RESULT: 42: I