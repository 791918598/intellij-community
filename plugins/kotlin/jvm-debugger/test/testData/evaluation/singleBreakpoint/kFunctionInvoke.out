LineBreakpoint created at kFunctionInvoke.kt:3
Run Java
Connected to the target VM
kFunctionInvoke.kt:3
Compile bytecode for 42.extInlineGenericFun()
Disconnected from the target VM

Process finished with exit code 0
